job-dsls/jobs/new_compile_downstream_build.groovy

```diff
@@ -0,0 +1,174 @@
+/**
+ * Creates compile downstream pullrequest (PR) jobs for kiegroup GitHub org. units.
+ * These jobs execute the downstream build skipping tests and skipping the gwt compilation
+ * for a specific PR to make sure the changes do not break the downstream repos.
+ */
+
+import org.kie.jenkins.jobdsl.Constants
+
+def final DEFAULTS = [
+        ghOrgUnit              : Constants.GITHUB_ORG_UNIT,
+        branch                 : Constants.BRANCH,
+        timeoutMins            : 650,
+        label                  : "kie-rhel7 && kie-mem16g",
+        ghAuthTokenId          : "kie-ci-token",
+        ghJenkinsfilePwd       : "kie-ci",
+        artifactsToArchive     : [
+                ",**/target/screenshots/**",
+                "**/target/kie-server-*ee7.war",
+                "**/target/kie-server-*webc.war",
+                "**/target/jbpm-server*dist*.zip",
+                "**/target/business-monitoring-webapp.war",
+                "**/target/business-central*wildfly*.war"
+        ]
+]
+// override default config for specific repos (if needed)
+def final REPO_CONFIGS = [
+        "lienzo-core"               : [],
+        "lienzo-tests"              : [],
+        "droolsjbpm-build-bootstrap": [],
+        "kie-soup"                  : [],
+        "appformer"                 : [],
+        "droolsjbpm-knowledge"      : [],
+        "drools"                    : [],
+        "optaplanner"               : [],
+        "jbpm"                      : [],
+        "kie-jpmml-integration"     : [],
+        "droolsjbpm-integration"    : [],
+        //"droolsjbpm-tools"          : [], // no other repo depends on droolsjbpm-tools
+        "kie-uberfire-extensions"   : [],
+        "kie-wb-playground"         : [],
+        "kie-wb-common"             : [],
+        "drools-wb"                 : [],
+        "optaplanner-wb"            : [],
+        "jbpm-designer"             : [],
+        "jbpm-work-items"           : [],
+        "jbpm-wb"                   : []
+        //"kie-wb-distributions"      : [] // kie-wb-distributions is the last repo in the chain
+]
+
+
+for (repoConfig in REPO_CONFIGS) {
+    Closure<Object> get = { String key -> repoConfig.value[key] ?: DEFAULTS[key] }
+
+    String repo = repoConfig.key
+    String repoBranch = get("branch")
+    String ghOrgUnit = get("ghOrgUnit")
+    String ghAuthTokenId = get("ghAuthTokenId")
+    String ghJenkinsfilePwd = get("ghJenkinsfilePwd")
+    String additionalLabel = get("label")
+    String additionalArticfacts = get("artifactsToArchive")
```

Comment:
Please fix the typo - `additionalArticfacts`

Reply:
@winklerm done